option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message HeroInstallAwakenEquipRequest
{
 required U64Type guid = 1;
 required uint32 pos = 2;
}
